src/operator/numpy/np_broadcast_reduce_op_value.cc

```diff
@@ -75,5 +75,56 @@ NNVM_REGISTER_OP(_backward_numpy_sum)
 .set_num_inputs(1)
 .set_attr<FCompute>("FCompute<cpu>", NumpyReduceAxesBackwardUseNone<cpu>);
 
+inline bool IsIntType(const int dtype) {
+  return (dtype >= 3);
```

Comment:
Better use enum variable names the than hard-coded number for readability and maintenance.

Reply:
Done.